src/main/java/uk/ac/ic/wlgitbridge/git/exception/SizeLimitExceededException.java

```diff
@@ -22,7 +29,7 @@ public String getMessage() {
     @Override
     public List<String> getDescriptionLines() {
         String filename =
-                path != null ? "File '" + path + "' is" : "There's a file";
+                path.isPresent() ? "File '" + path + "' is" : "There's a file";
```

Comment:
Does this need to be `path.get()`? When I try this on http://www.javarepl.com/ I get
```
java> "a" + Optional.ofNullable("foo") + "b"
... "aOptional[foo]b"
```